src/Tizen.NUI/src/public/Color.cs

```diff
@@ -230,8 +250,18 @@ public float G
         /// <summary>
         /// The blue component.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Color(...) constructor.
```

Comment:
```suggestion
        /// The setter is deprecated in API8 and will be removed in API10. Use the new Color(...) constructor.
```

Reply:
thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118